src/dataset/nuscenes_dataset.py

```diff
@@ -0,0 +1,22 @@
+from nuscenes import NuScenes
+from nuscenes.utils.splits import create_splits_scenes
```

Comment:
nuscenes is a third-party library. Please add a `requirements.txt` (or equivalent) with a listing of all your dependencies and the versions that your code needs.

Reply:
done